cmd/snap/cmd_find.go

```diff
@@ -118,6 +135,18 @@ func (x *cmdFind) Execute(args []string) error {
 		x.Positional.Query = ""
 	}
 
+	// section will be:
+	// - "show-all-sections-please" if the user specified --section
+	//   without any argument
+	// - "no-section-specifie" if "--section" was not specified on
```

Comment:
Super-nitpick: "specifie" :blush: . Feel free to ignore

Reply:
*cough* thank you!